src/types/emitters.ts

```diff
@@ -27,7 +27,7 @@ export type EmitterEvents =
 	| 'unnmute'
 	| 'captionsHide'
 	| 'setPlaybackRate'
-	| 'timeUpdate'
+	| 'timeupdate'
```

Comment:
What's the reason to rename this? Pascal-case seemed to be more in line with the other options (e.g. `fullscreenEnter`)

Reply:
The right native DOM event for time updating is `timeupdate`. ReactPlayer would work "partially" with "`timeUpdate`" (otherwise we would create 2 events : `timeUpdate` and `timeupdate`, that wont be interconnected as it should ).
It was a mistake to write it as `timeUpdate`. 
https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/timeupdate_event